blockstack/blockstackd.py

```diff
@@ -3482,6 +3499,82 @@ def run_blockstackd():
            print "Failed to sign snapshot"
            sys.exit(1)
 
+    elif args.action == 'export_migration_json':
+        path = str(args.path)
+        rc = fast_sync_import(working_dir, path, public_keys=None, num_required=0, verbose=True, delete_file=False)
+        if not rc:
+           print 'fast_sync failed'
+           sys.exit(1)
+
+        # treat this as a recovery
+        setup_recovery(working_dir)
+
+        # allow consensus_hash to be provided as a string or within a file
+        consensus_hash = args.consensus_hash
+        if os.path.exists(consensus_hash):
```

Comment:
I'm curious why the consensus hash would need to be in a file?  It's a 40-character string that could easily be passed through `argv`.

Reply:
If a file is provided then it's read, if not the 40-char string is used. The previous fast dump step exports the consensus hash in a file. Should this be restricted to only using only a string?